examples/styled-components/src/components/Example/Example.js

```diff
@@ -0,0 +1,16 @@
+
+import { decl } from 'bem-react-core';
+import styled from 'styled-components';
+
+import addStyle from '../../styleHOC';
+
+const style = Component => styled(Component)`
```

Comment:
By the way will severSiderRendering work with such declaration?

Reply:
It's example on create-react-app, it doesn't work with ssr.